app/templates/base/head.html

```diff
@@ -10,16 +10,17 @@
 <meta charset="utf-8">
 <meta name="theme-color" content="#1700b3">
 <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
-<meta name="author" content="UAVCAN Development Team">
+<meta name="author" content="OpenCyphal Development Team">
 
 <meta property="og:type"      content="website">
-<meta property="og:site_name" content="Uncomplicated Application-layer Vehicular Computing And Networking">
-<meta property="og:title"     content="Uncomplicated Application-layer Vehicular Computing And Networking">
+<meta property="og:site_name" content="Open technology for real-time intravehicular distributed computing and communication based on modern networking standards">
+<meta property="og:title"     content="Open technology for real-time intravehicular distributed computing and communication based on modern networking standards">
```

Comment:
This is gobbledygook. How about just "Open technology for real-time communications in cyber-physical systems." ?

Reply:
Fixed